check.py

```diff
@@ -107,6 +107,10 @@
             cmd += ['-O0'] # test that -O0 does nothing
         else:
           cmd += ['-O']
+        if 'PRINT_FULL' in asm:
```

Comment:
Don't feel strongly enough about it to block

Actually a best-of-both-worlds thing would be to leave the `PRINT_FULL` substring check, and rename the test to `PRINT_FULL_generic.asm.js` or something. That way both are reasonably-well supported, and it's more clear that generic tests go there.